cmd/ef_tests/levm/report.rs

```diff
@@ -133,21 +132,21 @@ pub fn summary_for_slack(reports: &[EFTestReport]) -> String {
         }}
     ]
 }}"#,
-        fork_summary_for_slack(reports, SpecId::PRAGUE),
-        fork_summary_for_slack(reports, SpecId::CANCUN),
-        fork_summary_for_slack(reports, SpecId::SHANGHAI),
-        fork_summary_for_slack(reports, SpecId::HOMESTEAD),
-        fork_summary_for_slack(reports, SpecId::ISTANBUL),
-        fork_summary_for_slack(reports, SpecId::LONDON),
-        fork_summary_for_slack(reports, SpecId::BYZANTIUM),
-        fork_summary_for_slack(reports, SpecId::BERLIN),
-        fork_summary_for_slack(reports, SpecId::CONSTANTINOPLE),
-        fork_summary_for_slack(reports, SpecId::MERGE),
-        fork_summary_for_slack(reports, SpecId::FRONTIER),
+        fork_summary_for_slack(reports, Fork::Prague),
+        fork_summary_for_slack(reports, Fork::Cancun),
+        fork_summary_for_slack(reports, Fork::Shanghai),
+        fork_summary_for_slack(reports, Fork::Homestead),
+        fork_summary_for_slack(reports, Fork::Istanbul),
+        fork_summary_for_slack(reports, Fork::London),
+        fork_summary_for_slack(reports, Fork::Byzantium),
+        fork_summary_for_slack(reports, Fork::Berlin),
+        fork_summary_for_slack(reports, Fork::Constantinople),
+        fork_summary_for_slack(reports, Fork::Paris),
```

Comment:
Move this between `Shanghai` and `Homestead` (it is an error for Paris to be this low in the list)